receiver/huaweicloudcesreceiver/integration_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build integration
```

Comment:
not sure this really falls into integration tests, since you're mocking the network calls...

on other receivers, i've seen implementations of integration tests really making use of separate processes and network stack.

since we don't have much knowledge on the server side of CES, i think this test level is ok, but we might need to call differently.

have you seen other examples of integration tests which just use mock clients on this repo?

Reply:
There are couple of examples using the same approach: 

- https://github.com/Huawei-IRC-Cloud-Telemetry-Engineering/opentelemetry-collector-contrib/blob/main/receiver/awscloudwatchreceiver/integration_test.go#L27
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/mongodbatlasreceiver/access_logs_integration_test.go#L28
-